### PR TITLE
BG-15531: Support bsv for non bitgo recoveries

### PR DESCRIPTION
**Pull request by glowang**
This PR adds mainnet support for BSV recoveries. The SDK component has been implemented here [https://github.com/BitGo/BitGoJS/pull/890](url)

### CODE REVIEW

File: src/components/non-bitgo.js [hunk 1]
```diff
@@ -117,7 +118,7 @@ class NonBitGoRecoveryForm extends Component {
         return obj;
       }, {});
 
-      if (this.state.coin === 'btc' && this.state.apiKey) {
+      if (this.state.coin === 'bsv' && this.state.apiKey) {
```
Comment (argjv) [inline src/components/non-bitgo.js:121]: Shouldn't it be both btc and bsv here?

File: src/constants/tooltips.js [hunk 2]
```diff
@@ -61,15 +61,15 @@ export default {
     apiKey: (coin) => {
       if (coin === 'eth' || coin === 'token') {
         return 'An Api-Key Token from etherscan.com required for Ethereum Mainnet recoveries';
-      } else if (coin === 'btc') {
-        return 'An Api-Key Token from blockchair.com required for Bitcoin Mainnet and Testnet recoveries';
+      } else if (coin === 'bsv') {
```
Comment (twtaylor) [inline src/constants/tooltips.js:64]: why did we remove the BTC tooltip?

File: src/components/non-bitgo.js (hunk 1 above)
Comment (glowang) [reply src/components/non-bitgo.js:121]: we no longer need it for btc, because we are using blockstream for btc now :)

**Review by argjv — APPROVED** [review]

File: src/components/unsigned-sweep.js [hunk 3]
```diff
@@ -217,13 +218,26 @@ class UnsignedSweep extends Component {
...
+      <Alert color='danger'>
+        <p>
+          Bitcoin SV Transactions are replayable on the Bitcoin Cash Network.
+        </p>
+        <p>
+          Please make sure you are the owner of the Destination Address to avoid
+          accidentally sending your BCH to an address you don't own.
+        </p>
+      </Alert>
+    }
```
Comment (twtaylor) [inline src/components/unsigned-sweep.js:235]: this looks good but note that at least in FE development, HTML typically uses double quotes for inline html

Comment (twtaylor) [reply src/components/unsigned-sweep.js:235]: this is more of a FE style thing than anything

File: src/components/unsigned-sweep.js [hunk 4]
```diff
@@ -167,7 +168,7 @@ class UnsignedSweep extends Component {
         return obj;
       }, {});
 
-      if (this.state.coin === 'btc' && this.state.apiKey) {
+      if (this.state.coin === 'bsv' && this.state.apiKey) {
```
Comment (twtaylor) [inline src/components/unsigned-sweep.js:171]: mmm again did we just drop support for btc through these flows? I don't understand the mutation of this

**Review by twtaylor — CHANGES_REQUESTED** [review]

File: src/constants/tooltips.js (hunk 2 above)
Comment (glowang) [reply src/constants/tooltips.js:64]: we no longer need it for btc, because we are using blockstream for btc now :)

**Pull request merged**